translations/Hindi/special-terms.md

```diff
@@ -0,0 +1,160 @@
+## Terms (शब्द)
+* **Message** : संदेश , Message in this case 
+is संदेश. There are other words like समाचार and 
+सूचना, but they mean news and information respectively. Besides, 
+संदेश is used in other contexts as well, and in all those contexts it 
+means message. , ```user``` को संदेश भेजें (Send message to 
+```user```)

```

Comment:
You need single backticks like this:

```
`hello-world`
```

for single line code blocks. I haven't gone through the full text yet so not marking as reviewed. Someone would take a look soon. Meanwhile, fix this markdown syntax. :smile:

Reply:
Noted, didn't know that feature was there